src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

```diff
@@ -402,7 +402,7 @@ Here are some additional distributed tracing UI details, rules, and limits:
     id="preserved-traces"
     title="Preserved traces"
   >
-    A preserved distributed trace is like an archived snapshot of the original trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is onlyavailable for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.
+    Preserved traces are similar to archived snapshots. They archive a trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is only available for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.
     
     However, there are some limitations to preserved traces:
```

Comment:
One last thing! Can we make this preface sentence more neutral? Maybe instead of a bulleted list, just saying 'Your preserved traces won't display span performance data or span anomaly data. If an entity... etc etc'.